src/GovUk.Education.ExploreEducationStatistics.Data.Model/Migrations/20240802123234_EES5368_UpdateRebuildIndexesSPToWriteRunDataToDB.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using GovUk.Education.ExploreEducationStatistics.Common.Extensions;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace GovUk.Education.ExploreEducationStatistics.Data.Model.Migrations
+{
+    /// <inheritdoc />
+    [ExcludeFromCodeCoverage]
+    public partial class EES5368_UpdateRebuildIndexesSPToWriteRunDataToDB : Migration
+    {
+        private const string PreviousRebuildIndexesMigrationId = "20240726153723";
```

Comment:
I would reference `EES5205_UpdateRebuildIndexesStoredProc.MigrationId` here rather than `20240726153723` so it can be clicked through to. That's the reason why we mark `MigrationId` as `internal` rather than `private` so that it can be referenced from the following migration.